compiler/rustc_feature/src/active.rs

```diff
@@ -688,6 +688,9 @@ declare_features! (
     /// not changed from prior instances of the same struct (RFC #2528)
     (incomplete, type_changing_struct_update, "1.58.0", Some(86555), None),
 
+    /// Allows rustdoc to automatically generate `#[doc(cfg(...))]`.
```

Comment:
"allows" sounds like you have to do some other step in addition to enabling the feature.
```suggestion
    /// Tells rustdoc to automatically generate `#[doc(cfg(...))]`.
```